src/components/AccountDrawer/MiniPortfolio/Activity/OffchainActivityModal.tsx

```diff
@@ -178,11 +194,13 @@ export function OrderContent({ order }: { order: SelectedOrderInfo }) {
       return (
         <ContentContainer>
           <ErrorContent />
-          <ThemedText.SubHeaderLarge>{t`Error`}</ThemedText.SubHeaderLarge>
+          <ThemedText.SubHeaderLarge>
+            <Trans>Error</Trans>
+          </ThemedText.SubHeaderLarge>
           <ThemedText.LabelSmall textAlign="center">
-            {t`Your swap couldn't be filled at this time. Try again or `}{' '}
+            <Trans>Your swap couldn&apos;t be filled at this time. Try again or </Trans>{' '}
             <LearnMoreLink href="https://support.uniswap.org/hc/en-us/articles/17515489874189">
-              {t` learn more`}.
+              <Trans>learn more.</Trans>
             </LearnMoreLink>
```

Comment:
Same issue here. Consider - if "learn more" should precede the other statement in a different language, it will not be phrased correctly.